packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -1489,6 +1495,60 @@ class _TimePickerInputState extends State<_TimePickerInput> {
   }
 }
 
+class _HourTextField extends StatelessWidget {
```

Comment:
It seems a little like overkill to factor out _HourTextField and _MinuteTextField, but OK.

Reply:
yeah I would have preferred not to, but it made it easier to properly test this.